opentelemetry-sdk/src/logs/simple_log_processor.rs

```diff
@@ -76,6 +76,7 @@ impl<T: LogExporter> SimpleLogProcessor<T> {
 
 impl<T: LogExporter> LogProcessor for SimpleLogProcessor<T> {
     fn emit(&self, record: &mut SdkLogRecord, instrumentation: &InstrumentationScope) {
+        let _suppress_guard = Context::enter_telemetry_suppressed_scope();
```

Comment:
Not related to the line of code, but still - we plan to _NOT_ add the suppression guard in the SimpleConcurrentLogProcessor, which means if the exporter attached to it is sending internal logs, it needs to add the suppression guard within the exporter? By default, ETW and user_events exporters will not be logging any debug/errors in their code?

Reply:
that is correct. SimpleConcurrentLogProcessor may need a rename to "ReentrantLogProcessor"/somethign else, and make it clear that it is to be used with exporters that do not cause re-entrancy back to itself.